validation/tcd_latest.xsd

```diff
@@ -50,6 +50,7 @@
       <xs:enumeration value="SharePoint"/>
       <xs:enumeration value="SAPBW"/>
       <xs:enumeration value="SwitchModal"/>
+      <xs:enumeration value="PasswordOnly"/>
```

Comment:
Update `AuthOptionEnum-ST` below as well with `Password`

Reply:
Made the change in the file.